### PR TITLE
DT-2295 Use the queue lib to provide configuration for topics

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

**Review by karenmillermoj — APPROVED** [review]
lgtm

File: src/main/resources/application-localstack.yml [hunk 1]
```diff
@@ -2,20 +2,23 @@ graceful:
   shutdown.enabled: false
 
 hmpps.sqs:
+  provider: localstack
   queues:
     prisoneventqueue:
       queueName: dps-queue
       dlqName: dps-dlq
+      subscribeTopicId: prisoneventtopic
+      subscribeFilter: '{"eventType":[ "PRISONER_RELEASED", "PRISONER_RECEIVED"] }'
```
Comment (andymarke) [inline src/main/resources/application-localstack.yml:11]: interestingly these filters are out of date, they should be the lower case versions. So I guess this isn't affecting the integration tests but when running locally (which I suspect nobody is doing)

**Review by andymarke — APPROVED** [review]
LGTM

File: src/main/resources/application-localstack.yml (hunk 1 above)
Comment (mikehalmamoj) [reply src/main/resources/application-localstack.yml:11]: Hmmm ok thanks.  I'll get back to you when I do my next changes :smile:

**Pull request merged**